galasa-parent/dev.galasa.framework.api.authentication/src/main/java/dev/galasa/framework/api/authentication/JwtWrapper.java

```diff
@@ -45,6 +45,11 @@ public JwtWrapper(HttpServletRequest req, Environment env) {
         usernameClaimOverrides = env.getenv(EnvironmentVariables.GALASA_USERNAME_CLAIMS);
     }
 
+    public JwtWrapper(String jwt, Environment env) {
```

Comment:
```    private static final List<String> DEFAULT_USERNAME_CLAIMS = List.of(
        "preferred_username",
        "name",
        "sub"
    );
    
```
We want to get rid of this. And if the user's claim names are not present then blow up with a decent error.